### DISCUSSION TITLE
which config I should change in order to increase memory capacity

### DISCUSSION THREAD

**Discussion by GuChangyang** [facebookincubator/velox#4765] [Q&A]
I got error and reported by below part of code : 

https://github.com/facebookincubator/velox/blob/e5c36aec30d997aadfb133c1263c4a2a1d3a14bc/velox/common/memory/MemoryPool.cpp#L774

out << "\nExceeded memory cap of " << succinctBytes(capacity_)
        << " when requesting " << succinctBytes(incrementBytes) << "\n";

I want to know which config should I change to increase this **capacity_**

thank you

**Comment by kgpai** [comment]
cc: @xiaoxmeng

**Comment by GuChangyang** [comment]
query.max-memory-per-node
